Modules/_randommodule.c

```diff
@@ -77,6 +77,10 @@
 #  include <process.h>            // getpid()
 #endif
 
+#ifdef MS_WINDOWS_NON_DESKTOP
```

Comment:
Yes it works fine to import it everywhere. In addition `<processthreadsapi.h>` is enough. It was failing since it did not find `GetCurrentProcessId`.